fibbingnode/misc/mininetlib/ipnet.py

```diff
@@ -48,7 +48,7 @@ def __init__(self,
                  private_ip_count=1,
                  private_ip_net='10.0.0.0/8',
                  controller_net='172.16.0.0/12',
-                 ipBase='192.168.0.0/16',
+                 ipBase='192.0.0.0/8',
```

Comment:
:+1: 
When I was creating a fat-tree topology with k > 4, the IPNet constructor was running out of broadcast domains... and this was just a quick fix when attempting to solve the problem.

What would be the best way to have more addresses? increasing the max prefix len?